### PR TITLE
chore(express): bump multer 1.4.4 to 1.4.4-lts.1 to fix "CVE-2022-24434"

### PR DESCRIPTION
**Pull request by sushant9096**
Signed-off-by: Sushant Zope <sushantzope9096@gmail.com>

## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/nestjs/nest/blob/master/CONTRIBUTING.md
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [x] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?
- [ ] Yes
- [x] No

## Other information

### CODE REVIEW

**Comment by sushant9096** [comment]
updated multer package version in packages/platform-express to fix vulnerability: [](https://github.com/advisories/GHSA-wm7h-9275-46v2)

**Comment by ShaharAdskAcc** [comment]
also waiting for it 👍

File: .gitignore [hunk 1]
```diff
@@ -34,3 +34,5 @@ yarn-error.log
 /packages/graphql
 /benchmarks/memory
 build/config\.gypi
+/packages/platform-express/builds/
+/packages/platform-express/dist/
```
Comment (kamilmysliwiec) [inline .gitignore:38]: Can you please revert this change?

Comment (sushant9096) [reply .gitignore:38]: ok i am going revert it

**Comment by hiagodotme** [comment]
Until this is released a workaround to resolve the issue is:

1. Delete node_modules folder and package-lock.json file
2. Use npm's override feature to rewrite the multer dependency to `1.4.4-lts.1`

Example:
```json
{
    ...
    "overrides": {
        "multer": "^1.4.4-lts.1"
    },
    "dependencies": {
    ...
}
```
> Remember to delete package-lock.json and node_modules and reinstall.

**Comment by pavleprica** [comment]
@hiagodotme just to expand that one, it has a requirement of `npm >= 8.3.0`. More on [overriding](https://docs.npmjs.com/cli/v8/configuring-npm/package-json#overrides)

**Comment by kamilmysliwiec** [comment]
lgtm

**Pull request merged**